src/App.css

```diff
@@ -1,39 +1,175 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
+#root {
+  background-color: #f5f6fa;
+  height: 100vh;
+}
```

Comment:
This is one of the top, neatly designed, and polished projects I have ever reviewed, bravo!:100::clap:

- [ ] However, your design shall meet the design in the Zeplin mock. A slight grey background over your main app page is missing. Please kindly add it:pray:

Zeplin's design:
<img width="1440" alt="Screen Shot 2021-11-06 at 21 02 28" src="https://user-images.githubusercontent.com/60043698/140616002-35f98830-08aa-4655-aff5-9d53a204c5d6.png">

Your design:
<img width="1440" alt="Screen Shot 2021-11-06 at 21 02 40" src="https://user-images.githubusercontent.com/60043698/140616011-decd3d95-2ff9-42b6-b3d5-ee6d4e125ba5.png">

Reply:
Thank you for your suggestions! But I don't understand what you require to add here. Do you mean the gray background behind the app?

Reply:
Hi @nachosala89 sorry if I wasn't so clear. And Yes, that grey background.
Please take a look at my [previously share Zeplin's snapshot](https://user-images.githubusercontent.com/60043698/140616002-35f98830-08aa-4655-aff5-9d53a204c5d6.png). I intentionally hovered over Zeplin's navbar so that those margin units from the top and sides can be visible.